src/app.md

```diff
@@ -725,6 +726,50 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given an array of unparsed version `&str`s, iterate and filter against a semver `VersionReq` and find the latest `Version`.
```

Comment:
I would give more details in the description. For example, explain the range is obtain via the `matches` and the `max`.

Reply:
Agreed. I'd remove the "an array" and "unparsed". I'd also add the hyperlinks to key identifiers like @ludwigpacifici suggests, Imho these would be `VersionReq::parse`, `VersionReq::matches` and optionally `Version::parse`.

Additionally "latest" is quite imprecise, I'd suggest rewriting it as 'satisfying the given range "<= 1.0.0"' .